creation/web_base/singularity_lib.sh

```diff
@@ -1647,6 +1647,14 @@ singularity_get_image() {
         [[ -z "$singularity_image" ]] && singularity_image=$(dict_get_first SINGULARITY_IMAGES_DICT)
     fi
 
+    if [[ -n "$CVMFS_MOUNT_DIR" ]]; then
+        # set things up here since the path needs to be bindmounted inside the container
+        local mount_home=${CVMFS_MOUNT_DIR/\/dist\/cvmfs/}
```

Comment:
if removing at the end, just %
if removing at the middle, just /
if removing at the start, just #

Reply:
CVMFS_MOUNT_DIR is set by cvmfsexec and also if someone else is mounting cvmfs somewhere else
things should work fine as long as CVMFS_MOUNT_DIR is set because singularity_lib handles some of this. test with singularity after mounting CVMFS in mode 1 or 3

Reply:
discussed about this and decided to remove the changes pertaining to singularity invocation with on demand cvmfs and include as a patch to the production series. this was decided since singularity invocation handling needs to be ensured irrespective of whether CVMFS is locally available where it could be mounted at `/cvmfs` or `/opt/cvmfs` or CVMFS is provisioned via _cvmfsexec_ using modes 1-3.